klippy/kinematics/delta.py

```diff
@@ -146,7 +146,17 @@ def check_move(self, move):
             limit_xy2 = -1.
         self.limit_xy2 = min(limit_xy2, self.slow_xy2)
     def get_status(self, eventtime):
-        return {'homed_axes': '' if self.need_home else 'xyz'}
+        return {
+            'homed_axes': '' if self.need_home else 'xyz',
+            'axis_minimum': {
+                'x': -math.sqrt(self.max_xy2),
+                'y': -math.sqrt(self.max_xy2),
+                'z': self.min_z},
+            'axis_maximum': {
+                'x': math.sqrt(self.max_xy2),
+                'y': math.sqrt(self.max_xy2),
+                'z': self.max_z}
```

Comment:
It would be preferable to calculate `sqrt(self.max_xy2)` once and then use that calculated value 4 times.